builtin/credential/ldap/path_config.go

```diff
@@ -143,8 +166,22 @@ func (b *backend) pathConfigWrite(
 	if groupdn != "" {
 		cfg.GroupDN = groupdn
 	}
+	groupfilter := d.Get("groupfilter").(string)
```

Comment:
OK. This is probably an OK breaking change in pursuit of a much better overall experience, but I'll need to ensure it is well documented.